### PR TITLE
revert _timestamp field name changes

### PR DESCRIPTION
**Pull request by fivetran-jamie**
**Are you a current Fivetran customer?** 

fivetran made PR

**What change(s) does this PR introduce?** 

- reverts some changes i made in https://github.com/fivetran/dbt_shopify_source/pull/40. i noticed we were changing some timestamp column names from `*_at` to `*_timestamp` and decided to not do this for consistency. however, this introduced quite a few downstream errors in the transform package (and shopify holistic reporting). i don't think it's worth it to make this change, so this PR reverts what i did

**Did you update the CHANGELOG?** 

- [x] Yes

**Does this PR introduce a breaking change?**

- [ ] Yes (please provide breaking change details below.)
- [x] No  (please provide an explanation as to how the change is non-breaking below.)

this AVOIDS a breaking change

**Did you update the dbt_project.yml files with the version upgrade (please leverage standard semantic versioning)? (In both your main project and integration_tests)** 

- [ ] Yes

**Is this PR in response to a previously created Bug or Feature Request**

- [ ] Yes, Issue/Feature [link bug/feature number here]
- [ ] No 

**How did you test the PR changes?** 

 

- [ ] Buildkite  
- [ ] Local (please provide additional testing details below)

**Select which warehouse(s) were used to test the PR**
 

- [ ] BigQuery
- [ ] Redshift
- [ ] Snowflake
- [ ] Postgres
- [ ] Databricks
- [ ] Other (provide details below)

**Provide an emoji that best describes your current mood**
 
🐖 

**Feedback**

We are so excited you decided to contribute to the Fivetran community dbt package! We continue to work to improve the packages and would greatly appreciate your [feedback](https://www.surveymonkey.com/r/DQ7K7WW) on our existing dbt packages or what you'd like to see next.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,12 +41,6 @@
 - Dependencies on `fivetran/fivetran_utils` have been upgraded, previously `[">=0.3.0", "<0.4.0"]` now `[">=0.4.0", "<0.5.0"]`.
 
 [PR #40](https://github.com/fivetran/dbt_shopify_source/pull/40) includes the following breaking changes:
-- In the following tables, `*_timestamp` columns have been renamed to `*_at` for consistency:
-  - `stg_shopify__customer`
-  - `stg_shopify__order`
-  - `stg_shopify__product_variant`
-  - `stg_shopify__product`
-  - `stg_shopify__transaction`
```
Comment (fivetran-avinash) [inline CHANGELOG.md:49]: Was there a reason we just changed these tables to the `timestamp` format? I see some others still in the `_at` format. https://github.com/fivetran/dbt_shopify_source/blob/feature/package-revamp/models/stg_shopify__refund.sql

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-jamie looks good, just some clarifying questions!

File: CHANGELOG.md [hunk 2]
```diff
@@ -41,12 +41,6 @@
 - Dependencies on `fivetran/fivetran_utils` have been upgraded, previously `[">=0.3.0", "<0.4.0"]` now `[">=0.4.0", "<0.5.0"]`.
 
 [PR #40](https://github.com/fivetran/dbt_shopify_source/pull/40) includes the following breaking changes:
-- In the following tables, `*_timestamp` columns have been renamed to `*_at` for consistency:
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply CHANGELOG.md:44]: yeah so we _currently_ rename these fields in main - https://github.com/fivetran/dbt_shopify_source/blob/main/macros/staging_columns.sql#L5

in my previous PR, i removed the alias when i moved around the staging column macros

File: CHANGELOG.md (hunk 1 above)
Comment (fivetran-jamie) [reply CHANGELOG.md:49]: not that i can tell, it seems like a stylistic choice Dylan made for the most pertinent timestamp columns

Comment (fivetran-avinash) [reply CHANGELOG.md:49]: Ah got it!

**Pull request merged**